web/src/ChatBox.js

```diff
@@ -57,12 +57,24 @@ class ChatBox extends React.Component {
     const newValue = this.state.value.replace(/<img src="([^"]*)" alt="([^"]*)" width="(\d+)" height="(\d+)" data-original-width="(\d+)" data-original-height="(\d+)">/g, (match, src, alt, width, height, scaledWidth, scaledHeight) => {
       return `<img src="${src}" alt="${alt}" width="${scaledWidth}" height="${scaledHeight}">`;
     });
+    const date = new Date();
```

Comment:
Use moment to format date, don't do it yourself

![image](https://github.com/casibase/casibase/assets/3787410/30ea6568-c75a-48e2-b835-e66d8d5bf23c)